bitnami/external-dns/values-production.yaml

```
@@ -411,6 +411,8 @@ rbac:
   ## Service Account for pods
   ## https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/
   ##
+  serviceAccount:
+    create: true
   serviceAccountName: default
```

Now, maybe it has more sense to create the `name` parameter as part of the `serviceAccount` object:
```suggestion
  serviceAccount:
    create: true
    name: default
```

And then use the new `serviceAccount.name` in those places where `serviceAccountName` was used.

PS: Same is applied to the _values.yaml_ and _values-production.yaml_